src/FontLib/Glyph/OutlineComposite.php

```diff
@@ -129,6 +129,11 @@ function parseData() {
 
       $this->components[] = $component;
     } while ($flags & self::MORE_COMPONENTS);
+    if ($flags & self::WE_HAVE_INSTRUCTIONS) {
+      $numInstr = $font->readUInt16();
+      $instr = $font->read($numInstr);
+      $this->components[count($this->components) - 1]->instructions = pack('n', $numInstr) . $instr;
+    }
```

Comment:
I think that MORE_COMPONENTS and WE_HAVE_INSTRUCTIONS should not coexist logically, but I cannot confirm the existence of such font files. If the code segment is placed inside a loop, the following code is synonymous with the original [pseudocode](https://learn.microsoft.com/en-us/typography/opentype/spec/glyf#code-try-0):
```
if (($flags & self::WE_HAVE_INSTRUCTIONS) && !($flags & self::MORE_COMPONENTS)) {
  $numInstr = $font->readUInt16();
  $instr = $font->read($numInstr);
  $instructions = pack('n', $numInstr) . $instr;
}
```
If any modifications are required, feel free to inform me.

Reply:
You're right I think the two flags together should not coexist. At least, the pseudocode makes that assumption and so I don't think the logic you wrote should change.

I do think we should move the instructions property from the OutlineComponent class to the OutlineComposite class. This change would more closely track the actual structure of the data, and it would also mirror how instructions are managed in the OutlineSimple class.

Reply:
If you want to place the code snippet inside a loop, please inform me again.

Reply:
I think we can keep the code where it is currently located.

Only change I think should be made is to move the instructions to the OutlineComposite class. But that suggestion is not directly related to the work you did in this PR so I'm fine making that change later myself.

Reply:
I believe it would be better if you made the changes yourself later on. Thank you.